indico/modules/events/management/forms.py

```diff
@@ -270,6 +267,23 @@ def _init_visibility(self, event):
             self.visibility.warning = _("This event's visibility is currently limited by that of '{}'.").format(
                 real_horizon.title)
 
+    def validate_permissions(self, field):
+        entries = [{
+            'principal': principal_from_fossil(entry[0], allow_emails=True, allow_networks=True),
+            'permissions': entry[1]
+        } for entry in field.data]
+        for e in entries:
+            principal = e['principal']
+            permissions = e['permissions']
+            if isinstance(principal, IPNetworkGroup):
+                if permissions != ['access']:
```

Comment:
While this works, I think using set logic would be nicer here: `if set(permissions) - {'access'}:`